### PR TITLE
Street View with 360° rotation (sensor)

### PR DESCRIPTION
**Pull request by paul-ndl**

### CODE REVIEW

**Review by jules20022000 — APPROVED** [review]
All the code makes sense. Really nice to have a sensor feature. Good job !

File: app/src/main/java/ch/epfl/sweng/hostme/ui/search/DisplayApartment.java [hunk 1]
```diff
@@ -73,6 +58,12 @@ public DisplayApartment() {
     public View onCreateView(@NonNull LayoutInflater inflater,
                              ViewGroup container, Bundle savedInstanceState) {
         root = inflater.inflate(R.layout.display_apartment, container, false);
+
+        Button maps_button = root.findViewById(R.id.maps_button);
+        maps_button.setOnClickListener(this::goToMapsFragment);
+        Button street_view_button = root.findViewById(R.id.street_view_button);
+        street_view_button.setOnClickListener(this::goToStreetViewFragment);
```
Comment (Scott2Williams) [inline app/src/main/java/ch/epfl/sweng/hostme/ui/search/DisplayApartment.java:65]: I see that you use `snake_case` for a lot of UI components. Personally, I have been using `camelCase` for all of my variables except `CAPITALIZED_CONSTANTS`. Maybe some time we can have a quick discussion to choose one as a standard, then I can make it consistent across the repo.

Comment (paul-ndl) [reply app/src/main/java/ch/epfl/sweng/hostme/ui/search/DisplayApartment.java:65]: yes of course we can see that together at the next meeting

File: app/src/main/java/ch/epfl/sweng/hostme/maps/MapsFragment.java [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+        try {
+            address = coder.getFromLocationName(this.fullAddress, 1);
+            Address location = address.get(0);
+            LatLng latlng = new LatLng(location.getLatitude(), location.getLongitude());
+            CameraPosition pos = new CameraPosition.Builder().target(latlng)
+                    .zoom(12.5f)
+                    .build();
+            googleMap.addMarker(new MarkerOptions()
+                    .position(latlng)
+                    .title("Your futur home!"));
```
Comment (Scott2Williams) [inline app/src/main/java/ch/epfl/sweng/hostme/maps/MapsFragment.java:70]: Fr-Eng typo: futur-future

**Review by Scott2Williams — APPROVED** [review]
Wow, great adaptation of the available resources/ sensors.  
> I left a couple of comments regarding syntax/ spelling, but since they don't affect performance, I didn't convert them into review conversations.

File: app/src/main/java/ch/epfl/sweng/hostme/maps/MapsFragment.java (hunk 2 above)
Comment (paul-ndl) [reply app/src/main/java/ch/epfl/sweng/hostme/maps/MapsFragment.java]: fixed!

**Pull request merged**